### ISSUE TITLE
The Last Of Us 2 - sce_sys/trophy/

### ISSUE THREAD

**Issue by brandonnuner** [hippie68/ftpdump#24]
I'm having an issue trying to dump The Last Of Us 2. I'm getting error "Wget reported error 139: "Unknown error (see https://www.gnu.org/software/wget/manual/html_node/Exit-Status.html." (partial logs below). If I look in the following folder (/mnt/sandbox/CUSA07820_000/app0/sce_sys/trophy) there is no trophy00.trp file. I'm not sure if that's the issue or not but figured I would mention it. 

sce_sys/about/right.sprx
53% --> CWD /mnt/sandbox/pfsmnt/CUSA07820-app0/sce_sys/keymap_rp
250 Requested file action okay, completed.
--> PASV
227 Entering Passive Mode (172,20,20,116,216,93).
--> LIST -a
150 File status okay; about to open data connection.
226 Closing data connection. Requested file action successful.
dr-xr-x--- 2 0 0 65536 Apr 24 2023 .
dr-xr-x--- 5 0 0 65536 Apr 24 2023 ..
sce_sys/keymap_rp/
53% --> CWD /mnt/sandbox/pfsmnt/CUSA07820-app0/sce_sys/trophy
250 Requested file action okay, completed.
--> PASV
227 Entering Passive Mode (172,20,20,116,216,94).
--> LIST -a
150 File status okay; about to open data connection.
226 Closing data connection. Requested file action successful.
dr-xr-x--- 2 0 0 65536 Apr 24 2023 .
dr-xr-x--- 5 0 0 65536 Apr 24 2023 ..
sce_sys/trophy/
53% Wget reported error 139: "Unknown error (see https://www.gnu.org/software/wget/manual/html_node/Exit-Status.html."
Error encountered, script aborted.

D:\ftpdump>

**Comment by hippie68** [comment]
It seems that Wget crashed due to a segmentation fault. I don't have an explanation. It could be a Wget bug or something interfering with its process. Does it always happen at that point? I'd try to update Wget, if possible (e.g. by updating the distro - WSL?), and see if it helps.
You can see the current Wget version by entering `wget --version`.

**Comment by brandonnuner** [comment]
Yes it was failing at the same place and I did attempt to update Wget which was already running latest code. I did a good amount of messing around yesterday and came to an assumption that the next step in the process was to extract the app.pkg file. So I decided today to start with the --dump command. Whether or not that was the smoking gun IDK but I was able to complete the process today. My steps are listed below. Everything just finished dumping appropriately. Time to setup the FPKG.

bash ftpdump.bash --dump CUSA07820 --app --patch --dlc -v --debug
bash ftpdump.bash -v --debug
      FTP Timed out during patch transfers. Reestablished FTP connection, removed -patch folder and ran below command.
bash ftpdump.bash -p -v --debug
